lib/blockly/javascript-commands.js

```diff
@@ -47,3 +47,17 @@ Blockly.JavaScript.position_y = function (block) {
   const position = block.getFieldValue('entity')
   return [`${position}.y`, Blockly.JavaScript.ORDER_NONE]
 }
+
+Blockly.JavaScript.random_number = function (block) {
+  const num1 = Blockly.JavaScript.valueToCode(block, 'NUM1', Blockly.JavaScript.ORDER_ATOMIC)
+  const num2 = Blockly.JavaScript.valueToCode(block, 'NUM2', Blockly.JavaScript.ORDER_ATOMIC)
+
+  const functionName = Blockly.JavaScript.provideFunction_(
```

Comment:
I think it would better to inject this function via the [robot-api](https://github.com/robotopia-x/robot-ide/blob/develop/models/runtime/robot-api.js#L17)